### ISSUE TITLE
[KalmanFilter] no tests (Trac #1120)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1114]
There are no tests for KalmanFilter.  There is a suspiciously named script called `resources/scripts/Test_Muon.py`.  If this is a test, it should be moved to the standard test directory `resources/test`.

Migrated from https://code.icecube.wisc.edu/ticket/1120
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "There are no tests for KalmanFilter.  There is a suspiciously named script called `resources/scripts/Test_Muon.py`.  If this is a test, it should be moved to the standard test directory `resources/test`.", 
    "reporter": "david.schultz", 
    "cc": "olivas", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[KalmanFilter] no tests", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-17T15:31:06", 
    "milestone": "", 
    "owner": "jacobi", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150818T15:42:07: 
* jacobi changed status from "new" to "closed"
* jacobi changed resolution from "" to "fixed"
* jacobi commented:

> In changeset 136406:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136406"
> Move test script to the appropriate directory. Closes #1114.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
